bigquery-writer/bigquery-writer.py

```diff
@@ -19,7 +19,7 @@
 SUBSCRIBER_NAME = "bq"
 KEYSPACE_NAME_INCOMING = "ilisten"
 KEYSPACE_NAME_UNIQUE = "ulisten"
-APP_CREDENTIALS_FILE = "bigquery-credentials.json"
+APP_CREDENTIALS_FILE = os.environ['GOOGLE_APPLICATION_CREDENTIALS']
```

Comment:
os.environ is a dictionary, if the env variable `GOOGLE_APPLICATION_CREDENTIALS` doesn't exist, this will fail with a `KeyError`. You should use `os.environ.get('GOOGLE_APPLICATION_CREDENTIALS')` and do your error detection by checking if this value is `None` in addition to it being a valid file